demo/app/app.ts

```diff
@@ -1,17 +1,17 @@
 ﻿import './bundle-config';
 import * as application from 'tns-core-modules/application';
-import { initTwilio } from 'nativescript-twilio';
+import { initTwilio, setupPushListener, setupCallListener} from 'nativescript-twilio';
 import { TwilioAppDelegate } from 'nativescript-twilio/delegate';
 
 // Please fill these values
 const accessTokenUrl = '';
+
 const accessTokenHeaders = {
-  // 'Authorization': 'Token sometoken',
+  // 'Authorization': 'Token '
```

Comment:
I think you committed this change by mistake. Please keep the `sometoken,` in order to give an example for app developers about how to use token auth.

Reply:
placed it back. yup i made a mistake here i used the hiveku tokens and forgot to place them back 👍